features/buyer/catalogue.feature

```diff
@@ -73,3 +73,12 @@ Scenario: User is able to search by keywords field on the search results page to
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
+
+Scenario: User is able to click a filter on the search results page to narrow down the results returned
+  Given I am on the /g-cloud page
```

Comment:
I think start directly on the `/g-cloud/search` page rather than starting a page before.